dist/index.js

```diff
@@ -46,7 +47,7 @@ window.DynamicFormsBuilderSidebar = BuilderSidebar;
  */
 const origFormioBuilder = Formio.builder;
 Formio.builder = function (element, form, options) {
-    options = options || { editForm: {} };
+    options = options || {project: process.env.MIX_APP_URL};
```

Comment:
Does this need to be mentioned in the docs/upgrade guide? I don't believe dynamic-forms required it before?

Reply:
@mc0918 I mean the MIX_APP_URL -- I don't think it used that before. If that's needed now, do we need to mention it in the docs/upgrade guide?

Reply:
I don't think we need it at all, I tried implementing it in CAP and it just caused a headache whereas just keeping editform: {} doesn't seem to cause any issues